### ISSUE TITLE
yum_repository is not available in ansible 2.0.2

### ISSUE THREAD

**Issue by Jack47** [ansible/ansible#15957]
##### ISSUE TYPE

- Bug Report
##### ANSIBLE VERSION

```
ansible 2.0.2.0
```
##### CONFIGURATION

  config file =
  configured module search path = Default w/o overrides
##### OS / ENVIRONMENT

Mac OS 10.9.5
##### SUMMARY

yum_repository is not available 
##### STEPS TO REPRODUCE

```
ansible  -i hosts.sample  -u root -m yum_repository all -vvvv
No config file found; using defaults
Loaded callback minimal of type stdout, v2.0
127.0.0.1 | FAILED! => {
    "failed": true,
    "msg": "The module yum_repository was not found in configured module paths. Additionally, core modules are missing. If this is a checkout, run 'git submodule update --init --recursive' to correct this problem."
}
```

##### EXPECTED RESULTS

yum_repository can be executed
##### ACTUAL RESULTS

```
127.0.0.1 | FAILED! => {
    "failed": true,
    "msg": "The module yum_repository was not found in configured module paths. Additionally, core modules are missing. If this is a checkout, run 'git submodule update --init --recursive' to correct this problem."
}
```

**Comment by danschmidt5189** [comment]
`yum_repository` was [added in 2.1](https://docs.ansible.com/ansible/yum_repository_module.html), which is why it's not available in 2.0. Ansible 2.1 is not available in PyPi (latest = 2.0.2.0), but you can install it [from source](http://docs.ansible.com/ansible/intro_installation.html#running-from-source).

**Comment by sivel** [comment]
Closing per the above explanation.
